tests/components/todoist/test_coordinator.py

```diff
@@ -0,0 +1,22 @@
+"""Tests for the TodoistCoordinator."""
```

Comment:
The design pattern is not to test classes directly, but instead exercise them through the integration APiI. This test should be removed.

If you'd like to exercise update failures it would be via the platforms e.g. calendar.

Reply:
You can likely revert the other test moves if the test harness no longer needs to be shared.